app/src/main/assets/ime/keyboard/org.florisboard.localization/extension.json

```
@@ -647,6 +647,20 @@
         "characters": "org.florisboard.layouts:urdu_phonetic"
       }
     },
+    {
+      "languageTag": "zh-CN-zhengma",
+      "composer": "org.florisboard.composers:appender",
+      "nlpProviders": {
+        "spelling": "org.florisboard.nlp.providers.han.shape",
+        "suggestion": "org.florisboard.nlp.providers.han.shape"
+      },
+      "currencySet": "org.florisboard.currencysets:yen",
+      "popupMapping": "org.florisboard.localization:en",
+      "preferred": {
+        "characters": "org.florisboard.layouts:qwerty",
+        "symbols": "org.florisboard.layouts:cjk",
+        "symbols2": "org.florisboard.layouts:cjk"
+
```

I'm currently not at home so cannot test it on my testing device but it seems that the JSON syntax got messed up while rebasing. (Will properly review the latest changes later)

```suggestion
    },
```

---

That's probably why subtype presets broke :thinking: